.changeset/perfect-meals-tie.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nomicfoundation/edr": patch
+---
+
+Added custom precompiles support for `eth_sendTransaction`
```

Comment:
Since we're treating this a bugfix, I'd probably phrase the changelog entry something like this: "Fixed custom precompiles not being applied in `eth_sendTransaction`. This enables RIP-7212 support in transactions.".